controller/server/controller.go

```diff
@@ -340,6 +327,28 @@ func (c *controller) DeleteServiceBinding(w http.ResponseWriter, r *http.Request
 	c.k8sStorage.DeleteServiceBinding(id)
 }
 
+// getBindingsFrom returns the set of bindings for a consuming service instance.
+//
+// Binding data is passed to the service broker right now as part of the
+// parameters in the form:
+//
+// parameters:
+//   bindings:
+//     <service-name>:
+//       <credential>
+func (c *controller) getBindingsFrom(sName string, fromBindings map[string]*scmodel.Credential) error {
+	bindings, err := c.k8sStorage.GetBindingsForService(sName, From)
```

Comment:
Nit, I find the `From` portion fairly difficult to read here -- I'd rather have qualifiers in names like `BindingDirectionFrom`.  This isn't an issue with this PR -- just noticed it during this review.